src/components/AutomationDetails/index.jsx

```diff
@@ -168,14 +168,23 @@ class AutomationDetails extends PureComponent {
     return appActivities.map((content, index) => (
       <div key={index}>
         <div className="automation-content">
-          <div className="content-row name">{content[contentData]}</div>
```

Comment:
Replacing this line with the `renderProjectTag` method causes the first column of the other tabs to be empty as this is what renders their content